tests/testthat/test-segmentcomparisontable.R

```diff
@@ -290,6 +290,65 @@ formSegmentation <- structure(c(2L, 6L, 8L, 8L, 4L, 9L, 6L, 3L, 2L, 4L, 1L, 7L,
 "30 to 34", "35 to 39", "40 to 44", "45 to 49", "50 to 54", "55 to 64",
 "65 or more"), questiontype = "PickOne", name = "d1", label = "Age", question = "Age")
 
+formOrderedCat <- list(Age = structure(c(2L, 6L, 8L, 8L, 4L, 9L, 6L, 3L, 2L, 4L,
```

Comment:
Can this and other variable definitions in this file be moved to rds files in inst/testdata? It is too long and hard to see which variables are being defined.

Reply:
I've moved the variable definitions to inst/testdata/segmentcomparisontabledata.rda